HW0/P5/P5.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+    x = np.arange(1,500)
+    y1 = x - 1
+    y2 = np.ceil(np.log2(x))
```

Comment:
I didn't use "ceil" function here. Do you mind elaborate your thought here for me?